src/api/admin.ts

```diff
@@ -40,7 +41,8 @@ type FirestoreVenueInput = Omit<
   mapIconImageUrl?: string;
 };
 
-export const createUrlSafeName = (name: string) => name.replace(/\W/g, "");
+export const createUrlSafeName = (name: string) =>
+  `${ROOT_URL}/v/${name.replace(/\W/g, "")}`;
```

Comment:
Why do we need this?

Reply:
for displaying the text 'The url of your venue will be sparkle.space/s/mylovelyvenue'. I noticed you are also using this function so will place the ROOT_URL in the caller.

Reply:
That's one more config variable we have to maintain when we could read it in the browser.

Reply:
hmm could do that i suppose